examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/variables.tf

```diff
@@ -1,21 +1,21 @@
-variable ocm_environment {
-    type = string
-    default = "production"
+variable "ocm_environment" {
+  type    = string
+  default = "production"
 }
 
-variable openshift_version {
-    type = string
-    default = "4.12"
+variable "openshift_version" {
+  type    = string
+  default = "4.12"
```

Comment:
I suggest to upgrade version to the latest GA 4.13

Reply:
So maybe better to remove this default, in order to enforce the user to add the version?
What do you say?